### ISSUE TITLE
Unexpected space-around-operator warning using @at-root %placeholder

### ISSUE THREAD

**Issue by polarstoat** [sasstools/sass-lint#1203]
What version of Sass Lint are you using?
```bash
$ sass-lint -V
1.12.1
```

Please include any relevant parts of your configuration
I use the default configuration, as can be found [here](https://github.com/sasstools/sass-lint/blob/master/lib/config/sass-lint.yml).

What did you do? Please include the actual source code causing the issue.
```scss
.icon {
  display: inline-block;
  margin-right: 5px;

  @at-root %normal-sprite {
    height: 16px;
    width: 16px;
  }

  &.smiley {
    @extend %normal-sprite;
  }
}

```

What did you expect to happen?
I expect not to see `space-around-operator` error when using placeholders (e.g. `%normal-sprite`) after `@at-root`.

What actually happened? Please include any error messages given to you by Sass Lint.
<img width="760" alt="screen shot 2018-05-23 at 16 19 56" src="https://user-images.githubusercontent.com/20626621/40434005-59b17618-5ea5-11e8-85ae-737d3b645170.png">

Received `space-around-operator` error on placeholder after `@at-root`.

If you're using a IDE plugin have you tried the CLI too?
<img width="620" alt="screen shot 2018-05-23 at 16 22 30" src="https://user-images.githubusercontent.com/20626621/40434104-89332652-5ea5-11e8-8587-e04145720c1a.png">

Yes, I use Sublime Text 3 with SublimeLinter and SublimeLinter-contrib-sass-lint, I get the same results when using a terminal window.

Placing a space before the `%placeholder` operator causes `node-sass` to fail. This seems like it could be a related issue to #1153.

**Comment by thykka** [comment]
I came across this same (/similar?) issue with the following markup:

```scss
$color-red: #f00;

.foo {
  @at-root a#{&}--highlight,
  &--highlight { // raises an error; space-around-operator
    color: $color-red;
  }
}

.bar {
  &--highlight { // does not raise an error
    color: $color-red;
  }
}

.baz {
  @at-root a#{&}--highlight { // does not raise an error
    color: $color-red;
  }
}
```

It seems like either of the selectors lint without errors by themselves, but combined they raise a lint error…
